editor/modes/visual-editor/block.js

```diff
@@ -320,6 +327,11 @@ class VisualEditorBlock extends Component {
 		// Generate a class name for the block's editable form
 		let { className = getBlockDefaultClassname( block.name ) } = blockType;
 		className = classnames( className, block.attributes.className );
+		// Don't show controls when collaboration is enabled.
+		if ( peerShowStyle ) {
```

Comment:
@gziolo @abhishekgahlot  wonder if the changes to the visual-block component could be abstracted in a HoC `withCollabEditing` to try to clean this file cleaner.

Reply:
That would be nice, I will explore this idea :)